source_code/backend/CS_department_monitor_backend/src/main/java/com/cs5324/monitorbackend/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.cs5324.monitorbackend.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
I like the idea of a global instead of Per controller like was in my setup, will add TODO labels and issues for refactoring if needed

Reply:
No changes on your end for this